autometa/config/__init__.py

```diff
@@ -33,7 +33,6 @@
 
```

Comment:
I don't know if it is important, but this file does not have `COPYRIGHT` on either side of the copyright notice.

Reply:
Also, I'm not sure if a general docstring is needed here to explain the contents of the file.